definitions/oi-kamon-agent/definition.yml

```diff
@@ -0,0 +1,18 @@
+domain: EXT
+type: SERVICE
+synthesis:
+  identifier: service.name
+  name: service.name
+
+  conditions:
+    - attribute: instrumentation.provider
+      value: kamon-agent
+
+  tags:
```

Comment:
This one doesn't need to be declared explicitly, we always inject it if it's present in the telemetry.

It doesn't hurt to leave it though.